frontend/src/components/Map/index.js

```diff
@@ -205,10 +207,39 @@ export default function Map(props, { draggable = true }) {
       },
       "waterway-label"
     );
+
+    map.on("mousemove", function (e) {
+      var countries = map.queryRenderedFeatures(e.point, {
+        layers: ["world-layer"],
+      });
+
+      if (countries.length > 0) {
+        const country_name = countries[0].properties.name;
+        const country = covidData.filter(
+          (country) => country.name === country_name
+        );
+
+        if (country.length > 0 && country[0].confirmed) {
+          document.getElementById("pd").innerHTML =
+            "<h2>" +
+            country_name +
+            "</h2><h3>" +
+            country[0].confirmed +
+            " cases confirmed</h3>";
+        } else {
+          document.getElementById("pd").innerHTML =
+            "<h2>" + country_name + "</h2><h3> NA </h3>";
+        }
+      } else {
+        document.getElementById("pd").innerHTML =
+          "<h2> Confirmed Cases </h2> <h3>Hover over a state or country!</h3>";
```

Comment:
Would it be better to change it to hover/click for mobile user? Looks really good!

Reply:
Ok